### ISSUE TITLE
Provide a way to show size of sub-elements of a given folder

### ISSUE THREAD

**Issue by jonhoo** [sharkdp/diskus#3]
One of the most common use-cases for `du -hs` by far for me (and also what `dust` caters to), is to do `du -hs *` to find the largest directories in a given directory (usually the current one). It doesn't seem like `diskus` can currently operate in that mode? It'd be awesome if that kind of "tell me what's large" quick-n-dirty mode could be supported somehow!

**Comment by sharkdp** [comment]
I was not planning to turn this into a full-blown replacement for `du`, but that is indeed also my most common use-case.

What about changing the usage to
```
diskus [path]...
```

where `diskus` would just show the total size of the current directory, but `diskus path1 path2` would show the total size for each path? The latter could simply be used with `diskus *`.

**Comment by archusr** [comment]
[ncdu](https://dev.yorhel.nl/ncdu) (NCurses Disk Usage) comes to mind – https://en.wikipedia.org/wiki/Ncdu

**Comment by jonhoo** [comment]
That seems perfect! I think it'd also be handy for the resulting list to be sorted by size, not by the order of the arguments, but with that that's pretty much all I'd need!

**Comment by polyzen** [comment]
Could just pipe it through `sort`.

**Comment by jonhoo** [comment]
@polyzen yup, you _can_ always do that, but I'd argue that since it's by far the most common use-case, it seems reasonable for that to be the default :) That is also what `dust` does.

**Comment by kpcyrd** [comment]
I usually use `du -sch *` which lists the size of each file/folder and the total size of all combined. This would be really useful to have.

**Comment by joeshaw** [comment]
I tend to use `du -h -d 1` over `du -sh *` for this, because it only lists directories.

It is sometimes nice to use a greater depth, like when trying to track down an unknown, particularly large subdirectory.  So, at the risk of feature creep, it'd be nice to have summaries of arbitrary depth.

**Comment by rNoz** [comment]
My main use case is exactly like @kpcyrd and @polyzen: per directory and file, total size of every listed file and output piped to `sort -h`.

**Comment by mw55309** [comment]
Hello, was this ever implemented?

**Comment by nu8** [comment]
Maybe it could be just used with `fd` instead?

https://github.com/sharkdp/fd

for example I tried this:

~~~
fd -x diskus
~~~

but its not quite right, as it only displays the sizes, where most people would want something like:

~~~
10 a.txt
20 b.txt
~~~

**Comment by busfahrer** [comment]
+1, from reading the description of this tool I assumed that's what it actually was - my 90% use case of for du is `du -s * | sort -n`. Would love to use diskus' great performance for that use case!

**Comment by LuzianHahn** [comment]
Hey, I just provided a PR for defaulting to not aggregating different paths together, which were provided at the same time to `diskus`. This should make it more similar to `du -sh *`, to identify problematic directories at a certain location. If this is not merged, I can also provide a respective build in my fork.
